spec/middleware/MiddlewareUtil.ts

```diff
@@ -13,8 +13,8 @@ import { appendRequestHeader, generateUUID, getRequestHeader, setRequestHeader }
 
 describe("MiddlewareUtil.ts", async () => {
 	describe("getRequestHeader", () => {
-		const key: string = "Content-Type";
-		const value: string = "application/json";
```

Comment:
which rule does this one violate.  i have mixed feelings as I somewhat like being explicit.

Reply:
@typescript-eslint/no-inferrable-types.